### DISCUSSION TITLE
EM7455 support china mobile

### DISCUSSION THREAD

**Discussion by isdqwq** [danielewood/sierra-wireless-modems#38] [General] [answered]
dear sir.
    how to support China Mobile , China Unicom and China Telecom?
below link
https://source.sierrawireless.com/resources/airprime/minicard/74xx/airprime-em_mc74xx-approved-fw-packages/#sthash.W8M0MTnH.dpbs

it looks only for USA Carrier Approved

**Comment by danielewood** [comment, marked as answer]
Chinese carriers use TDD-LTE, the only TDD band the 7455 supports is Band 40. It may function on Band40, assuming the Chinese carriers dont lock out IMEIs like Sprint does in the US. 

As for Carrier Approval of a firmware, that will never happen in China as this modem isnt intended for the Chinese market. My script uses the Generic firmware, which doesnt lock out any carriers, but that doesnt mean that the Chinese carriers wont block the device.

**Comment by Rajiv-Roy** [comment]
Dear sir,
Many thanks for such a great tutorial and script for making the EM7455 module with a wifi router. I have successfully updated generic firmware on my Dell EM7455 and it is working great. How can I enable Band 40 which is 2300 Mhz  LTE-TDD 40 on this EM7455 module? Any possibilities to enable the same by terminal AT commands. I am from India and all mobile service providers in India support 2CA carrier aggregation on LTE-TDD band 40. 
Many thanks.
